packages/main/src/plugin/kubernetes/kubernetes-port-forward-service.ts

```diff
@@ -144,36 +120,17 @@ export class KubernetesPortForwardService implements IDisposable {
   /**
    * Deletes an existing forward configuration.
    * @param config - The forward configuration to delete.
-   * @param mapping - The mapping to delete, if mapping is undefined, delete all
    * @returns Void if the operation successful.
    * @see UserForwardConfig
    */
-  async deleteForward(config: UserForwardConfig, mapping?: PortMapping): Promise<void> {
-    const keys: string[] = [];
-    if (mapping) {
-      keys.push(this.getPortForwardKey(config, mapping));
-    } else {
-      keys.push(...config.forwards.map(forward => this.getPortForwardKey(config, forward)));
-    }
-
-    for (const key of keys) {
-      const disposable = this.#disposables.get(key);
-      if (disposable) {
-        disposable.dispose();
-        this.#disposables.delete(key);
-      }
-    }
-
-    const newConfig: UserForwardConfig = {
-      ...config,
-      forwards: config.forwards.filter(forward => !keys.includes(this.getPortForwardKey(config, forward))),
-    };
-
-    if (newConfig.forwards.length === 0) {
-      await this.configManagementService.deleteForward(config);
-    } else {
-      await this.configManagementService.updateForward(config, newConfig);
-    }
+  async deleteForward(config: UserForwardConfig): Promise<void> {
+    const key = this.getPortForwardKey(config);
+
+    const disposable = this.#disposables.get(key);
```

Comment:
suggestion: remove key and getPortForwardKey method

```suggestion
    const disposable = this.#disposables.get(config.id);
```

Reply:
👍